plugin/v2/plugin.go

```diff
@@ -241,6 +284,15 @@ func (p *Plugin) InitSpec(s specs.Spec, libRoot string) (*specs.Spec, error) {
 		s.Mounts = append(s.Mounts, m)
 	}
 
+	defaultMounts := make([]specs.Mount, 0, len(s.Mounts))
+	for _, m := range s.Mounts {
+		if !strings.HasPrefix(m.Destination, "/dev/") && true { // TODO: && user specified /dev
```

Comment:
How about delete the mount with "/dev/" prefix from s.Mounts rather than making copies?

for index, _ := range s.Mounts {
		if strings.HasPrefix(m.Destination...) {
			s.Mounts = append(s.Mounts[:index], s.Mounts[index+1:]...)
		}
	}